<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOTedarikciDuzenleme</class>
 <widget class="QWidget" name="TMOTedarikciDuzenleme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tedarikçi Ekleme Formu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../resimler/tedarikci.jpg</normaloff>
    <normalon>../../../resimler/tedarikci.jpg</normalon>../../../resimler/tedarikci.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblTedarikciAdi">
     <property name="text">
      <string>Tedarikçi Adı</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leTedarikciAdi"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblTedarikciAdresi">
     <property name="text">
      <string>Tedarikçi Adresi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTedarikciTelefonu">
     <property name="text">
      <string>Tedarikçi Telefonu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leTedarikciTelefon">
     <property name="inputMask">
      <string>\0 (D99) 999 99 99</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblTedarikciYetkiliKisi">
     <property name="text">
      <string>Tedarikçi Yetkili Kişi</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leTedarikciYetkiliKisi"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnKaydet">
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>438</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnIptal">
     <property name="text">
      <string>İptal</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="pteTedarikciAdresi"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
